apps/web/src/views/Home/components/Banners/NewIFOBanner.tsx

```diff
@@ -0,0 +1,251 @@
+  right: 0;
+  bottom: 0px;
+  z-index: 1;
+  overflow: hidden;
+  border-radius: 32px;
+  ${({ theme }) => theme.mediaQueries.sm} {
+    overflow: visible;
+  }
+  > img:first-child {
+    position: absolute !important;
+    bottom: -40px;
+    right: -50px;
+    z-index: 2;
+    width: 201px;
+    ${({ theme }) => theme.mediaQueries.sm} {
+      right: 10px;
+      bottom: 0px;
+      width: 278px;
+    }
+    ${({ theme }) => theme.mediaQueries.md} {
+      width: 330px;
+      right: 60px;
+      bottom: 0px;
+    }
+  }
+  > img:nth-child(2) {
+    position: absolute !important;
+    bottom: -2px;
+    right: 75px;
+    z-index: 3;
+    width: 115px;
+    ${({ theme }) => theme.mediaQueries.sm} {
+      bottom: 0px;
+      right: 235px;
+    }
+    ${({ theme }) => theme.mediaQueries.md} {
+      width: 148px;
+      bottom: 0px;
+      right: 340px;
+    }
+  }
+`
+
+const StyledSubheading = styled.div`
+  padding-right: 100px;
+  position: relative;
+  font-family: 'Kanit';
+  font-style: normal;
+  font-weight: 800;
+  font-size: 24px;
+  line-height: 98%;
+  letter-spacing: 0.01em;
+  font-feature-settings: 'liga' off;
+  margin-left: 3px;
+  margin-top: 5px;
+  margin-bottom: 5px;
+  color: #08060b;
+  ${({ theme }) => theme.mediaQueries.md} {
+    font-size: 40px;
+    padding-left: 0px;
+    margin-top: 10px;
+    &::after {
+      padding-right: 0px;
+    }
+  }
+`
+
+export const CountDownWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: self-start;
+  background-color: #e8a571;
+  font-family: Kanit;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 90%;
+  color: #08060b;
+  padding: 6px;
+  border-radius: 8px;
+  margin-top: 5px;
+  gap: 0px;
+  flex-direction: column;
+  ${({ theme }) => theme.mediaQueries.sm} {
+    flex-direction: row;
+    gap: 8px;
+    padding: 8px;
+    font-size: 20px;
+    line-height: 110%; /* 22px */
+  }
+`
+
+const BackGroundCircle = styled.div`
+  position: absolute;
+  top: 41px;
+  right: -117px;
+  width: 296px;
+  height: 296px;
+  border-radius: 296px;
+  opacity: 0.5;
+  background: radial-gradient(50% 50% at 50% 50%, #c87c59 0%, rgba(228, 150, 91, 0) 100%);
+  z-index: 1;
+  filter: blur(64.5px);
+`
+
+const BackGroundCircle2 = styled.div`
+  position: absolute;
+  top: -148px;
+  left: calc(50% - 148px);
+  width: 296px;
+  height: 296px;
+  border-radius: 296px;
+  background: radial-gradient(50% 50% at 50% 50%, #d4e3e0 0%, rgba(212, 227, 224, 0) 100%);
+  filter: blur(49.5px);
+`
+
+const BackGroundCircle3 = styled.div`
+  position: absolute;
+  bottom: -150px;
+  left: calc(50% - 238px);
+  width: 296px;
+  height: 296px;
+  border-radius: 296px;
+  background: radial-gradient(50% 50% at 50% 50%, #beb1ce 0%, rgba(190, 177, 206, 0) 100%);
+  filter: blur(64.5px);
+`
+
+export function Countdown() {
+  const { t } = useTranslation()
+  const { isMobile } = useMatchBreakpoints()
+  const countdown = useCountdown(1704376800)
```

Comment:
Wrong time. Should be this check IFO config.
`1704369600, // Thu Jan 04 2024 12:00:00 UTC`